### PR TITLE
testing

### PR DESCRIPTION
**Pull request by manibollu**
hey please review I made changes to the ami @prudhviadapa

### CODE REVIEW

File: ec1.yml [hunk 1]
```diff
@@ -5,5 +5,6 @@ Resources:
   MyEC2Instance:
     Type: AWS::EC2::Instance
     Properties:
-      ImageId: ami-0ff8a91507f77f867
       InstanceType: t1.micro
+ # testing pr
+
```
Comment (prudhviadapa) [inline ec1.yml:10]: Why do we need this please delete this empty space and rest LGTM @manibollu

Comment (manibollu) [reply ec1.yml:10]: ok prudhvi will resolve it

Comment (prudhviadapa) [reply ec1.yml:10]: ok

Comment (prudhviadapa) [reply ec1.yml:10]: ok

**Review by prudhviadapa — APPROVED** [review]

**Pull request merged**